homeassistant/components/icloud/config_flow.py

```diff
@@ -243,30 +246,57 @@ async def _show_trusted_device_form(
             errors=errors or {},
         )
 
-    async def async_step_verification_code(self, user_input=None):
+    async def async_step_verification_code(self, user_input=None, errors=None):
```

Comment:
We should inline the forms in the step methods instead. Step methods only take a single parameter, the user input.